crates/bevy_asset/src/server/loaders.rs

```diff
@@ -0,0 +1,152 @@
+use crate::{loader::ErasedAssetLoader, AssetLoader};
+use bevy_log::warn;
+use bevy_tasks::IoTaskPool;
+use bevy_utils::HashMap;
+use futures_lite::Future;
+use std::{any::TypeId, fmt::Debug, sync::Arc};
+
+/// Storage for [`AssetLoader`]'s, providing helper methods for efficient access.
+#[derive(Debug, Default)]
+pub struct AssetLoaders {
+    values: HashMap<TypeId, MaybeAssetLoader>,
```

Comment:
Could we find a more descriptive field name?

Names are hard, but perhaps

- `type_id_to_asset_loader`

or perhaps you have other ideas.

Reply:
No that's a good name, I just took the name from the original structure, back when that value was just a `Vec`